tests/data/dataset_readers/wikitables/wikitables_dataset_reader_test.py

```diff
@@ -16,15 +16,96 @@ def test_reader_reads(self):
                            "a", "part", "of", "the", "usl", "a", "-", "league", "?"]
         assert [t.text for t in instance.fields["question"].tokens] == question_tokens
         entities = instance.fields['table'].knowledge_graph.get_all_entities()
-        assert len(entities) == 47
-        assert 'fb:row.row.year' in entities
+        assert len(entities) == 77
+        assert sorted(entities) == [
+                # The table cell entity names.  Duplicates have trailing _2, _3, etc.
+                'fb:cell.10_727',
+                'fb:cell.11th',
+                'fb:cell.11th_2',
+                'fb:cell.1st',
+                'fb:cell.1st_round',
+                'fb:cell.1st_round_2',
+                'fb:cell.1st_western',
+                'fb:cell.2',
+                'fb:cell.2001',
+                'fb:cell.2002',
+                'fb:cell.2003',
+                'fb:cell.2004',
+                'fb:cell.2005',
+                'fb:cell.2006',
+                'fb:cell.2007',
+                'fb:cell.2008',
+                'fb:cell.2009',
+                'fb:cell.2010',
+                'fb:cell.2_10',
+                'fb:cell.2_2',
+                'fb:cell.2_3',
+                'fb:cell.2_4',
+                'fb:cell.2_5',
+                'fb:cell.2_6',
+                'fb:cell.2_7',
+                'fb:cell.2_8',
+                'fb:cell.2_9',
+                'fb:cell.2nd',
+                'fb:cell.2nd_pacific',
+                'fb:cell.2nd_round',
+                'fb:cell.3rd_pacific',
+                'fb:cell.3rd_round',
+                'fb:cell.3rd_round_2',
+                'fb:cell.3rd_round_3',
+                'fb:cell.3rd_usl_3rd',
+                'fb:cell.4th_round',
+                'fb:cell.4th_round_2',
+                'fb:cell.4th_western',
+                'fb:cell.5_575',
+                'fb:cell.5_628',
+                'fb:cell.5_871',
+                'fb:cell.5th',
+                'fb:cell.6_028',
+                'fb:cell.6_260',
+                'fb:cell.6_851',
+                'fb:cell.7_169',
+                'fb:cell.8_567',
+                'fb:cell.9_734',
+                'fb:cell.did_not_qualify',
+                'fb:cell.did_not_qualify_2',
+                'fb:cell.did_not_qualify_3',
+                'fb:cell.did_not_qualify_4',
+                'fb:cell.did_not_qualify_5',
+                'fb:cell.did_not_qualify_6',
```

Comment:
Ok, I'll change it back.  I've also since realized that sempre doesn't do this, after looking at a bunch more data.  I'm not not convinced it's correct, though...

Reply:
"Did not qualify" occurring in multiple rows is still the same value, right?

Reply:
It is now.  See my comment above.  I've put this back to how it was, only detecting duplicates for columns.